.github/ISSUE_TEMPLATE/---say-thank-you.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "❤️ Say thank you"
+about: "Tell us how you use ecs-cli \U0001F60D"
+title: ''
+labels: Love
```

Comment:
Is it really an issue🤷‍♂️?

Reply:
😄 it's inspired by https://github.com/nock/nock/issues/new/choose to get feedback on how folks use the tool